pkg/dockerregistry/server/quotarestrictedblobstore.go

```diff
@@ -72,5 +80,44 @@ type quotaRestrictedBlobWriter struct {
 func (bw *quotaRestrictedBlobWriter) Commit(ctx context.Context, provisional distribution.Descriptor) (canonical distribution.Descriptor, err error) {
 	context.GetLogger(ctx).Debug("(*quotaRestrictedBlobWriter).Commit: starting")
 
+	if err := admitBlobWrite(ctx, bw.repo, provisional.Size); err != nil {
+		context.GetLogger(ctx).Error(err.Error())
```

Comment:
Ahh, I must have lost it during rebase. Thanks once again!

On 05/26/2016 10:28 AM, Maciej Szulik wrote:
>
> In pkg/dockerregistry/server/quotarestrictedblobstore.go 
> <https://github.com/openshift/origin/pull/8195#discussion_r64708302>:
>
> > @@ -72,5 +80,44 @@ type quotaRestrictedBlobWriter struct {
> >  func (bw *quotaRestrictedBlobWriter) Commit(ctx context.Context, provisional distribution.Descriptor) (canonical distribution.Descriptor, err error) {
> >  	context.GetLogger(ctx).Debug("(*quotaRestrictedBlobWriter).Commit: starting")
> >
> > +	if err := admitBlobWrite(ctx, bw.repo, provisional.Size); err != nil {
> > +		context.GetLogger(ctx).Error(err.Error())
>
> You're still double-logging, don't log here at all, since the ones in |admitBlobWrite| are already more descriptive.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/openshift/origin/pull/8195/files/4dd8bedae408abe3dfa677ccb1cecb31601918dc..fe4fdc248771a925aa32397378455c41ed3c32ce#r64708302>
>